src/storage/global/globalSlice.ts

```diff
@@ -94,9 +94,7 @@ const globalSlice = createSlice({
       .addCase(languageChangeStarted, (state) => toggleLoading(state, true, true))
       .addCase(languageChangeFinished, (state) => toggleLoading(state, false, true))
       .addCase(walletSaved, (state, action) => {
-        const { id, name, encrypted } = action.payload.wallet
-
-        state.wallets.push({ id, name, encrypted })
+        state.wallets.push(action.payload.wallet)
```

Comment:
Before this change, the `wallets` state was an array of wallet objects like this:
```ts
{
  id: string
  name: string
  encrypted: string
}
```

With the change you made, we are storing extra unnecessary data in this array:

```
{
  id: string
  name: string
  encrypted: string

  lastUse: Date

  mnemonic: string
  passphrase?: string
}
```

I think we should simply extract the `lastUse` one and not store the rest (mnemonic, passphrase).